components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenManagementDAOImpl.java

```diff
@@ -725,6 +733,7 @@ public void revokeSaaSTokensOfOtherTenants(String consumerKey, String userStoreD
             ps.setString(3, OAuthConstants.TokenStates.TOKEN_STATE_ACTIVE);
             ps.setString(4, consumerKey);
             ps.setInt(5, tenantId);
+            ps.setInt(6, IdentityTenantUtil.getLoginTenantId());
```

Comment:
`ps.setInt(5, tenantId);` -> what's the tenantId passed here?

Reply:
This method (`revokeSaaSTokensOfOtherTenants`) revokes SaaS tokens of other tenant. It is the tenant of the token and since SaaS, could be different from the app tenant